blivet/devices/stratis.py

```diff
@@ -0,0 +1,87 @@
+# devices/stratis.py
+#
+# Copyright (C) 2020  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Vojtech Trefny <vtrefny@redhat.com>
+#
+
+import logging
+log = logging.getLogger("blivet")
+
+from .storage import StorageDevice
+from ..static_data import stratis_info
+from ..size import Size
+from ..storage_log import log_method_call
+from .. import devicelibs
+
+
+class StratisPoolDevice(StorageDevice):
+    """ A stratis pool device """
+
+    _type = "stratis_pool"
+    _resizable = False
+    _packages = ["stratisd"]
+    _dev_dir = "/stratis"
```

Comment:
I developed and tested this on F33, this needs to be changed to `/dev/stratis` for F34.